server/src/ClientToDBAPI/CountiesRoute/mainRoute.ts

```diff
@@ -31,26 +70,4 @@ countiesRoute.get('/county/displayName', adminMiddleWare, (request: Request, res
         });
 });
 
-countiesRoute.get('/allCounties', (request: Request, response: Response) => {
```

Comment:
מחקת את הroute הזה ומשתמשים בו

Reply:
it used in landing page investigation table to get the list of counties for swutch counties select

Reply:
fixed